cypress/integration/page.spec.js

```diff
@@ -721,7 +760,7 @@
       });
 
       it('should not show NewCommentForm', () => {
-        newCommentForm.el().should('not.exist')
+        newCommentForm.el().should('not.exist');
```

Comment:
There is a missing semicolon at the end of the `cy.intercept` statement. Ensure that each statement is properly terminated to avoid syntax errors.